SwiftTweaks/Tweak.swift

```diff
@@ -76,6 +79,24 @@ extension Tweak where T: SignedNumber {
 	}
 }
 
+extension Tweak {
+	// This can't be an initializer because generic types apparently can't handle fixed-type initializers
+	public static func stringList(_ collectionName: String, _ groupName: String, _ tweakName: String, options: [String], defaultValue: String?=nil) -> Tweak<StringOption> {
+		assert(!options.isEmpty, "Options list cannot be empty (stringList tweak \"\(tweakName)\")")
+		if defaultValue != nil && options.index(of: defaultValue!) == nil {
```

Comment:
This condition can be the first parameter of `precondition` instead of condition + failure.